src/adapters/github.less

```diff
@@ -73,6 +73,7 @@
 
     .octotree_treeview {
       .octotree_header_repo {
+        width: 152px;
```

Comment:
Not so great when ppl resize the sidebar, right?

Reply:
Re-size the sidebar? How would someone do that?

Reply:
Would you prefer

```css
width: 66%;
```

or do you have another option?

![githubextension](https://user-images.githubusercontent.com/10472448/32574621-9e7c3aee-c486-11e7-8be1-0bbcf6b677e4.gif)